dts/bindings/pinctrl/alif,pinctrl.yaml

```
@@ -0,0 +1,131 @@
+# Copyright (c) 2025 Alif Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+title: Alif Semiconductor pin controller
```

```suggestion
title: Pin controller
```
The vendor name is visible in the compatible string.
This will make the name shorter in the generated Board Features table in the doc.